src/java/map/BinaryMask.java

```diff
@@ -93,6 +96,41 @@ public void set(int x, int y, boolean value) {
         mask[x][y] = value;
     }
 
+    public float getDistanceBetweenPoints(int x1, int y1, int x2, int y2) {
```

Comment:
This function shouldn't go here as it is not related to BinaryMask. It should go in utils in some class. One should really just use Vector2f but if you really want a function for just the primitives then put a function in utils